datasource.js

```diff
@@ -6,13 +6,22 @@ export class WalloraAPI extends RESTDataSource {
     this.baseURL = "https://wallora-api.herokuapp.com/api/";
   }
 
-  async getUserSettings(v1AccessToken, v2AccessToken) {
+   async getUserSettings(v1AccessToken, v2AccessToken) {
```

Comment:
do not need a space here at the begining

Reply:
resolved